web/modules/custom/portland/modules/portland_openid_connect/src/Plugin/Action/EnableUser.php

```diff
@@ -45,10 +46,15 @@ public function execute($account = NULL)
    */
   public function access($object, AccountInterface $account = NULL, $return_as_object = FALSE)
   {
-    /** @var \Drupal\user\UserInterface $object */
-    $access = $object->status->access('edit', $account, TRUE)
-      ->orIf($object->access('update', $account, TRUE));
-
-    return $return_as_object ? $access : $access->isAllowed();
+    // The update permission's name could be either Update or Edit, we check both and allow access if either one is allowed.
+    // Please note the Core API AccessResult::orIf will return Forbidden if either access result is Forbidden
```

Comment:
I'm not sure what this comment means or what it's referring to because it doesn't look like you're actually using the orIf() in the code now.

Reply:
Comments updated.